tableauserverclient/models/tableau_auth.py

```diff
@@ -1,5 +1,6 @@
 class TableauAuth(object):
-    def __init__(self, username, password, site=None, site_id='', user_id_to_impersonate=None):
+    def __init__(self, username='', password='', site=None, site_id='', user_id_to_impersonate=None, token_name='',
+                 personal_access_token=''):
```

Comment:
Why default to `''` instead of `None` for these params? Admittedly we're a little inconsistent, but `None` won't silently get serialized if there's a bug

Reply:
The pattern should be to create a different object for personal access tokens.  TableauAuth is specifically for Tableau username/password login.  You want to create a PersonalTokenAuth or similarly named object.

Reply:
Implemented the new approach on PR 465